pkg/jobrunaggregator/jobrunaggregatorapi/types.go

```diff
@@ -2,6 +2,9 @@ package jobrunaggregatorapi
 
 import "time"
 
+// For all structs that end in "Row", they correspond to a Big Query row in a similarly
+// named table.  See https://console.cloud.google.com/bigquery?project=<theProject>
+// and pick a dataset in use to see the table.
```

Comment:
This is probably technically an invalid godoc comment, I believe they're supposed to start with the name of the type your comment is attached to. (i.e. UnifiedTestRunRow corresponds to...) Some linters might reject this, and perhaps you could work around that with a newline before the typedef starts, or moving up to package level documentation. See: https://go.dev/blog/godoc